src/CollectionTrait.php

```diff
@@ -40,7 +40,7 @@ public function distinct()
     /**
      * Returns a lazy collection with items from all $collections passed as argument appended together
      *
-     * @param \Traversable[]|array ...$collections
+     * @param (array|\Traversable)[] ...$collections
```

Comment:
I have never seen this format of type doc, could you stick to what was here before and maybe fix it to `array[]|\Traversable`?